### DISCUSSION TITLE
Move the boundary for central void formation

### DISCUSSION THREAD

**Discussion by xuxiaobei1995** [idaholab/moose#17657] [Q&A General] [answered]
Hi all, 

     I'm working on a 2D-RZ cylindrical problem in which a void caused by the temperature gradient gradually forms at the center of the cylinder.  So my question is that is there any way to move the centerline boundary to simulate the central void formation? Any suggestion is appreciated.
![image](https://user-images.githubusercontent.com/34068460/115491067-e1f7c080-a291-11eb-8265-a711c34169c8.png)

**Comment by coskrrb2002** [comment]
Dear xuxiaobei1995,

I think you can do it with activation of element.

You should read this post: #16008 .


Best regards,
Nakkyu

**Reply by xuxiaobei1995** [reply]
Hi Nakkyu,

I read the documentation of the ActivateElementUserObject. It seems that the object can only activate the elements. But what I need is to de-activate the elements... Anyway, thanks for your reply!

**Comment by hugary1995** [comment, marked as answer]
You can use this: [https://mooseframework.inl.gov/source/userobject/CoupledVarThresholdElementSubdomainModifier.html](https://mooseframework.inl.gov/source/userobject/CoupledVarThresholdElementSubdomainModifier.html). If you don't add kernels on a subdomian, then that subdomain is effectively "deactivated".

**Reply by xuxiaobei1995** [reply]
Hi Gary,

Thanks for your suggestion! It is just what I need. I will try it for my problem.
